src/resources/js/tickets-registration-page.js

```diff
@@ -538,6 +540,25 @@ window.tribe.tickets.registration = {};
 			}
 		}
 
+		// Validation for Tribe Horizontal Date Picker
+		if ( $input.hasClass( 'tribe-horizontal-date-picker-value' ) ) {
+			const wrapper = $input.closest( '.tribe-horizontal-date-picker-container' );
+			const day = wrapper.find('.tribe-horizontal-date-picker-day');
+			const month = wrapper.find('.tribe-horizontal-date-picker-month');
+			const year = wrapper.find('.tribe-horizontal-date-picker-year');
+
+			[ day, month, year ].forEach( el => {
+				// Check if given value is a positive number, even if it's a string
+				if ( isNaN( parseInt( el.val() ) ) || parseInt( el.val() ) <= 0 ) {
+					el.addClass( 'ticket-meta__has-error' );
+
+					isValidfield = false;
```

Comment:
I would also break out of this foreach so we don't have to run through the other fields if we don't need to.

Reply:
I don't think we can break out of it, because if month or year is invalid, we would like to flag both to the user.

Reply:
Oh gotcha, fair enough.